core/device/model/Device.py

```diff
@@ -128,7 +128,7 @@ def checkSecret(self, secret: str) -> bool:
 
 
 	def _loadConfigs(self):
-		displayName = self._data.get('displayName', None)
+		displayName = self._deviceConfigs.get('displayName', None)
 		if not displayName or displayName.lower() == 'none':
```

Comment:
good catch, it's because I removed the display name column from the DB, of course it's not i ndata anymore!!